RetroBar/Languages/magyar.xaml

```diff
@@ -20,13 +22,15 @@
     <s:String x:Key="allow_font_smoothing">_Betűsimítás engedélyezése</s:String>
     <s:String x:Key="collapse_tray_icons">Az _értesítési terület ikonjainak összecsukása</s:String>
     <s:String x:Key="show_clock">Mutasd az _órát</s:String>
+    <s:String x:Key="show_multi_mon">Show on _multiple displays</s:String>
```

Comment:
@webmaster442, could you update your translation, there are three lines missing, and in line 13, it refers to the "Themes" folder, not a theme folder.

Reply:
Yes. Should I create a separate merge request for it ?

Reply:
As you prefer, I do this so that a request does not have to be made, but if you want to make the changes under your name, it would be preferable that you do so.

Reply:
Thank you!
_This is now resolved by #179_.